tests/installation/bootloader_s390.pm

```diff
@@ -294,15 +293,12 @@ sub format_dasd {
         die "dasdfmt died with exit code $r" unless (defined($r) && $r == 0);
     }
 
-    # until Agama get better UI for DASD, activation will be done via parmfile (bsc#1238891#c9)
-    unless (is_agama) {
-        # bring DASD down again to test the activation during the installation
-        if (script_run("timeout --preserve-status 20 bash -x /sbin/dasd_configure $dasd_path 0") != 0) {
-            record_soft_failure('bsc#1151436');
-            script_run('dasd_reload');
-            assert_script_run('dmesg');
-            assert_script_run("bash -x /sbin/dasd_configure -f $dasd_path 0");
-        }
+    # bring DASD down again to test the activation during the installation
+    if (script_run("timeout --preserve-status 20 bash -x /sbin/dasd_configure $dasd_path 0") != 0) {
+        record_soft_failure('bsc#1151436');
+        script_run('dasd_reload');
+        assert_script_run('dmesg');
+        assert_script_run("bash -x /sbin/dasd_configure -f $dasd_path 0");
```

Comment:
same here, we don't want to bring it down, it should be down and untouched, we leave all the responsability to Agama to to this, so we can test that Agama does on its own, Agama will run similar commands under the hood.